### PR TITLE
10712, 10677 - Fixed focus indicators for both main toolbar and cell …

### PR DESCRIPTION
**Pull request by halerankin**
…toolbar.

This PR addresses #10712 and #10677

I revised masked-icon styles to use the :before pseudo element. That way, existing focus styles will display as intended. This affects the main notebook toolbar and the cell toolbar. This works across all three themes. Screenshots below show only the light theme focus.

image

image

image

### CODE REVIEW

File: src/sql/media/icons/common-icons.css [hunk 1]
```diff
@@ -546,7 +548,7 @@ Includes non-masked style declarations. */
 }
 
 .book.codicon {
-
+	-webkit-mask-image: url("book_image.svg");
```
Comment (Charles-Gagnon) [inline src/sql/media/icons/common-icons.css:551]: You have this twice now?

Comment (halerankin) [reply src/sql/media/icons/common-icons.css:551]: @Charles-Gagnon @alanrenmsft Let's get this focus fix in and settle on colors next.

Comment (halerankin) [reply src/sql/media/icons/common-icons.css:551]: Fixed

**Comment by halerankin** [comment]
@Charles-Gagnon @alanrenmsft 
Step one: get focus working. Done per this PR. 
Step two: select colors that provide suitable contrast and emphasis for all three themes.

**Comment by alanrenmsft** [comment]
@halerankin, can we add some space between the button content and focus outline? for example  there is no space between the +Cell dropdown icon and the border. also looks like we should add some margin for the buttons in the toolbar so that the button focus indicator and the toolbar border are not overlapping.

**Pull request merged**